tests/gtests/test_SW_Markov.cc

```diff
@@ -57,14 +57,12 @@ namespace {
     EXPECT_DOUBLE_EQ(0., SW_All.Markov.cfnw[0]);
     EXPECT_DOUBLE_EQ(0., SW_All.Markov.cfnd[0]);
 
-    // Reset to previous global state
-    // Reset_SOILWAT2_after_UnitTest();
     SW_MKV_deconstruct(&SW_All.Markov);
   }
 
 
   // Check seeding of RNG for weather generator
-  TEST(WGTest, Seeding) {
+  TEST_F(AllTest, Seeding) {

```

Comment:
Please change `TEST_F(AllTest, Seeding)` to something like `TEST_F(AllTest, WeatherGeneratorRNGSeeding)`